src/models/generation/thermal_generation.jl

```diff
@@ -39,18 +39,16 @@ Data Structure for thermal generation technologies subjecto to seasonality const
     Examples
 
 """
-struct ThermalGenSeason <: ThermalGen
+struct ThermalGenSeason <: ThermalGen # TODO: Do we need this? Removal of scalingfactor makes it a duplicate of above
```

Comment:
Removing makes sense to me. It shouldn't cause any harm unless someone else is depending on it.

Reply:
Done